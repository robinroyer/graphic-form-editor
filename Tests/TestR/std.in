R r01 10 10 5 5
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
MOVE r01 1 1
LIST
UNDO
LIST
REDO
LIST
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
LIST
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
REDO
LIST
EXIT