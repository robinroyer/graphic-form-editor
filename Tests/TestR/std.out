OK
#New object: r01
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
OK
#Object sucessfully moved
R r01 32 32 27 27
OK
#Last command successfully 'undo'ed
R r01 31 31 26 26
OK
#Last command successfully 'redo'ed
R r01 32 32 27 27
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
R r01 12 12 7 7
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'redo'ed
R r01 32 32 27 27
